src/pages/[platform]/start/quickstart/index.mdx

```diff
@@ -2378,7 +2378,7 @@ Running this command will scaffold Amplify backend files in your current project
 
 To deploy your backend use Amplify's per-developer cloud sandbox. This feature provides a separate backend environment for every developer on a team, ideal for local development and testing. To run your application with a sandbox environment, you can run the following command:
 
-```bash
+```bash title="Terminal" showLineNumbers={false}
```

Comment:
Might be worth making a github issue for a component to replace these in the future.

Reply:
filed https://github.com/aws-amplify/docs/issues/7394